### ISSUE TITLE
Question cannot be completed by keyboard alone

### ISSUE THREAD

**Issue by deltanetdan** [deltanet/adapt-ppq-audio#17]
This prevents keyboard and screen reader users progressing. 

![PPQ-picture 1](https://user-images.githubusercontent.com/4161219/120504869-6757b000-c3bc-11eb-9e7f-f6acf5712a65.png)
